libbeat/outputs/elasticsearch/client.go

```diff
@@ -320,7 +325,18 @@ func createEventBulkMeta(
 		ID:       id,
 	}
 
+	if opType == opTypeDelete {
+		if id != "" {
+			return eslegclient.BulkDeleteAction{Delete: meta}, nil
+		} else {
+			err := fmt.Errorf("op_type delete requires _id")
+			return nil, err
```

Comment:
Nit: since you're not really using any formatting specifiers, you could just user `errors.New` here. Also, could you probably just combine these two statements into one.

```suggestion
			return nil, errors.New("op_type delete requires _id")
```

Reply:
Alternate nit: you could keep the `fmt.Errorf` and use `opTypeKey`.

```suggestion
			return nil, fmt.Errorf("%v delete requires _id", opTypeKey)
```